pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -129,10 +121,6 @@ def pre_optimizer_step(
         """
         # apex amp does not support closures.
         lambda_closure()
-
-        if not pl_module.automatic_optimization:
```

Comment:
Did we decide to remove the hooks in manual optimization ?

Reply:
Nope, will be kept. These changes will be reverted